Authorization/Database/PermissionSomebodyGroup.cs

```diff
@@ -10,10 +10,8 @@ namespace Starcounter.Authorization.Database
     [Database]
     public class PermissionSomebodyGroup : Relation
     {
-        [SynonymousTo(nameof(WhatIs))]
```

Comment:
By simply removing the attribute you are creating a new independent property, which is not correct. Please see my property implementations in Simplified.